### DISCUSSION TITLE
Unable to Connect NCP node to the network using 5 volt power adopter.

### DISCUSSION THREAD

**Discussion by danishkhan708** [openthread/openthread#10658] [Q&A]
Hello everyone,

I am currently working on an OpenThread project to create a mesh network consisting of 9 nodes. My goal is to analyze the network's efficiency and latency.

Hardware Setup:
1. nRF52833 DK as a Border Router, flashed with `ot-rcp.hex`
2. nRF52840 USB Dongle as an End Node/Router, flashed with `ot-ncp-ftd.hex`
3. Raspberry Pi 3B
4. Linux Machine

Current Setup:

- The Border Router is connected to the Raspberry Pi, acting as the network leader.
- The other nodes (nRF52840 USB Dongles) function as End Nodes or Routers.
- To connect to an NCP on the serial port, I use `WPANTUND` and execute the following command on the Raspberry Pi or Linux machine:

  sudo /usr/local/sbin/wpantund -o NCPSocketName /dev/ttyACM1 -o WPANInterfaceName wpan1
 
- After running this command, the NCP node joins the network, and I can successfully ping all the nodes.

Problem:
I want to deploy the nodes at a distance from each other, so I plan to power the nRF52840 USB Dongles using a 5V power adapter. However, when I do this, the nodes do not join the network because they require the `wpantund` command to be run on the Raspberry Pi. This becomes a significant challenge, especially as the number of nodes increases (e.g., up to 40), since it is not feasible to connect each node to a separate Raspberry Pi.

Question:
Is there a solution to this problem? How can I deploy the nRF52840 USB Dongles using only a power adapter without needing to connect each one to a Raspberry Pi or Linux machine for network joining?

I would greatly appreciate any help or suggestions. Thanks in advance!

**Comment by jwhui** [comment]
If you are looking to setup a Thread network, I would suggest taking a look at one of the codelabs, for example:
- [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router)

Note that `wpantund` is not really supported anymore. Also, for devices that you want to operate standalone, you'll need to have a firmware build that supports that.
